### ISSUE TITLE
demo1.json failure at runtime

### ISSUE THREAD

**Issue by mgoeminne** [cetic/rts-gen#6]
When executing the CLI with demo1.json, an error occurs at runtime, because a "origin" attribute is missing.

**Comment by alexnuttinck** [comment]
If you have this Exception: Tsimulus: java.util.NoSuchElementException: key not found: origin

It's due to the "origin" argument which is missing in your .json file.

For instance, adding the following line "origin": "2016-01-01 12:34:56.789" after the "timestep" argument will fix the error.

The following .json file is the [demo1.json](https://github.com/cetic/TSimulus/blob/master/example/demo1.json) updated with the "origin" argument:

```json
{
  "generators" : [
    {
      "name": "daily-generator",
      "type": "daily",
      "points": {"10:00:00.000": 4, "17:00:00.000": 32}
    },
    {
      "name": "noisy-daily",
      "type": "aggregate",
      "aggregator" : "sum",
      "generators": [
        "daily-generator",
        {
          "type": "arma",
          "model": { "phi": [0.5], "std": 0.25, "c" : 0, "seed": 159357},
          "timestep": 180000,
          "origin": "2016-01-01 12:34:56.789"
        }
      ]
    },
    {
      "name":  "partial-daily",
      "type": "partial",
      "generator" : "daily-generator",
      "from": "2016-01-01 00:00:00.000",
      "to": "2017-01-01 00:00:00.000"
    }
  ],
  "exported" : [
    {
      "name": "series-A",
      "generator": "daily-generator",
      "frequency": 60000
    },
    {
      "name": "series-B",
      "generator": "noisy-daily",
      "frequency": 30000
    }
  ],
  "from" : "2016-01-01 00:00:00.000",
  "to" : "2016-10-01 00:00:00.000"
}

**Comment by mgoeminne** [comment]
Fixed with e56973cc2df796023b73f78b8f5c5287ce29ec96 that includes the configuration file proposed by @alexnuttinck
